apps/graphql/src/apps/booking/queries/__tests__/ConfirmPaymentMutation.test.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+
+import { generateTestsFromFixtures } from '@kiwicom/test-utils';
```

Comment:
The idea is to have one test file read all fixtures. It should not be necessary to create this file again within the same folder there is already a `queries.test.js` here. Now, I guess this file will also be moved to mutations folder, so it is ok.